tools/codegen/dest/lazy_ir.py

```diff
@@ -0,0 +1,387 @@
+from typing import List, Optional, Union, Tuple
+import itertools
+from typing_extensions import Literal
+from dataclasses import dataclass
+import textwrap
+from tools.codegen import local
+from tools.codegen.context import method_with_native_function, native_function_manager
+from tools.codegen.utils import Target, mapMaybe
+from tools.codegen.model import (BaseType, OptionalType, DispatchKey, NativeFunction,
+                                 NativeFunctionsGroup, SchemaKind, FunctionSchema,
+                                 TensorOptionsArguments, ListType,
+                                 DeviceCheckType, Argument, assert_never,
+                                 is_cuda_dispatch_key, BackendIndex,
+                                 gets_generated_out_inplace_wrapper, OperatorName,
+                                 Arguments, SelfArgument, Return)
+from tools.codegen.api.types import (BaseTy, BaseCppType, BaseCType, OptionalCType,
+                                     Binding, ConstRefCType, NamedCType,
+                                     CppSignature, CppSignatureGroup,
+                                     Expr, MutRefCType, kernel_signature,
+                                     DispatcherSignature, VectorCType, intT, ListCType,
+                                     scalarT, scalarTypeT)
+import tools.codegen.api.meta as meta
+import tools.codegen.api.cpp as cpp
+import tools.codegen.api.structured as structured
+from tools.codegen.api.translate import translate
+from tools.codegen.selective_build.selector import SelectiveBuilder
+
+valueListT = BaseCppType('std', 'vector<ir::Value>')
+valueT = BaseCppType('ir', 'Value')
+intArrayT = BaseCppType('std', 'vector<int64_t>')
```

Comment:
technically this is kinda redundant; I think you should be able to use `VectorCType(BaseCType(intT))` from `types.py`